Help/Hackathon.rst

```diff
@@ -0,0 +1,196 @@
+  skip this section.
+
+Grabbing coala
+~~~~~~~~~~~~~~
+
+Grabbing coala is easy. Supposing you have pip installed (Python package
+manager), you can simply run:
+
+::
+
+  pip3 install coala-bears
+
+This will grab for you coala and all its' plugins.
+
+Using coala
+~~~~~~~~~~~
+
+Okay, so you got coala, now what? What does it do, how do I use it?
+
+coala is a static code analysis tool. In a nutshell, it fixes/detects issues
+in your code, such as syntax problems, formatting, semantic.
+
+coala is a tool that is used by command line arguments. It requires a
+configuration. So unless you have a configuration file, running ``coala`` will
+not be enough. Let's see what basic arguments you can give it.
+
+What is a bear?
+A bear is a plugin which runs for a routine in your code.
+As an example, we have IndentationBear which is a bear you can run to check
+whether your code is indented correctly.
+
+Let's first check if there's an existing bear for the language you have
+written your project in. This can be done by running:
+
+::
+
+  coala --show-bears --filter-by-language C
+
+This will list all bears available for C. You can replace "C" with your
+used language. Now, supposing there are more choices of bears and you are
+totally unsure what to use, simply visit
+`this link <https://github.com/coala/bear-docs#all>`__
+and click on each bear under your language's header. In a matter of minutes
+you should decide which bear to use.
+
+
+Now you are in your project's directory, have coala installed and know
+what bears you want to run. Here we learn about the two core arguments that
+we use to run coala: ``--files`` and ``--bears``. They are probably intuitive,
+from what their names' say.
+
+An example of a command which runs ``SpaceConsistencyBear`` on all ``C`` files
+in the current directory is:
+
+::
+
+  coala --files=*.c --bears=SpaceConsistencyBear
+
+Great, now coala should have yielded some results (supposing your code wasn't
+perfect), and they should prompt you in the terminal and ask you for actions.
+If you want to fix your code, you can give it actions (such as applying the
+patches found), but for the purpose of this tutorial simply seeing coala run
+should suffice.
+
+Contributing to coala
+---------------------
+
+Now you should be here in case you have completed the first step (running
+coala) or you have previously used coala.
+
+coala uses Github and Gitlab to maintain its' repositories. During this
+hackathon we will only enlist the 3 most important repositories because
+these have accessible issues.
+
+coala is split in three main repositories:
+ - coalib, which is the core of the coala with its' libraries
+ - coala-bears, which holds basically each bear, grouped on languages
+ - documentation, which has the documentation (even the page you are on now)
+
+For the issue you will find, you will have to clone locally the correspoding
+repository and work on its' code.
```

Comment:
This can be better stated as 

> To resolve an issue, you will have to locally clone the corresponding respository and work on its code.

Also, note that correspoding -> corresponding